.travis.yml

```diff
@@ -41,6 +41,11 @@ matrix:
       os: linux
       dist: xenial
       env: COVERAGE=true ZIPALIGN_INSTALL=true PHPUNIT_FLAGS="-v -c phpunit.xml --coverage-clover=coverage.clover"
+      
+    - php: 8.0
+      os: linux
+      dist: xenial
+      env: COVERAGE=true ZIPALIGN_INSTALL=true PHPUNIT_FLAGS="-v -c phpunit.xml --coverage-clover=coverage.clover"
```

Comment:
For only one version, code coverage should be considered. I usually leave the cover on the latter.
Remove coverage from php7.4.